### PR TITLE
Accessible color picker

### PR DESCRIPTION
**Pull request by joshua-larks**

### CODE REVIEW

File: src/styles/components/Inputs.less [hunk 1]
```diff
@@ -1,30 +1,39 @@
...
+    border-color: @light-5;
+  }
 
   &:active,
   &:focus {
     box-shadow: none;
-    border-color: @light-5;
-    background-color: @day-input-bg;
+    border: 2px solid @dark-2;
+    // background-color: @day-input-bg;
```
Comment (cevicher0) [inline src/styles/components/Inputs.less]: can you remove this ?

Comment (joshua-larks) [reply src/styles/components/Inputs.less]: Removed.

File: src/components/ColorPicker.vue [hunk 2]
```diff
@@ -6,21 +6,31 @@
       :placeholder="placeholder"
       @click="showPicker()"
       @input="updateFromInput"
+      :class="{ 's-colorpicker__input--error': error }"
     />
+    <div v-if="error" class="s-colorpicker____input-error">
```
Comment (cevicher0) [inline src/components/ColorPicker.vue]: should be __ ??
s-colorpicker___input-error

File: src/components/ColorPicker.vue [hunk 3]
```diff
@@ -90,6 +103,7 @@ export default class ColorPicker extends Vue {
 
   documentClick(e: any) {
     let el = this.$refs.colorpicker;
+    console.log(el);
```
Comment (cevicher0) [inline src/components/ColorPicker.vue]: do you need to keep console.log here?

File: src/components/NewFeatureOverlay.vue [hunk 4]
```diff
@@ -163,8 +163,12 @@ export default class NewFeatureOverlay extends Vue {
 }
 </script>
 
-<style lang="less" scoped>
+<style lang="less">
```
Comment (cevicher0) [inline src/components/NewFeatureOverlay.vue]: do we have to remove scoped ?

File: src/components/TextInput.vue [hunk 5]
```diff
@@ -313,10 +287,10 @@ export default class TextInput extends Vue {
...
 }
 
-.s-form-field__label--error,
-.s-form-field__error-text {
-  color: @dark-red;
-}
+// .s-form-field__label--error,
+// .s-form-field__error-text {
+//   color: @dark-red;
+// }
```
Comment (cevicher0) [inline src/components/TextInput.vue]: can you remove this ?

File: src/components/TextInput.vue [hunk 6]
```diff
@@ -351,16 +325,16 @@ export default class TextInput extends Vue {
...
     color: @day-title;
-    font-weight: 500;
   }
 
-  .s-form-field__input:focus + .s-form-field__label--error {
-    color: @dark-red;
-  }
+  // .s-form-field__input:focus + .s-form-field__label--error {
+  //   color: @dark-red;
+  // }
```
Comment (cevicher0) [inline src/components/TextInput.vue]: can you remove this ?

**Pull request closed without merging**